src/main/java/Main.java

```diff
@@ -8,6 +8,78 @@ public static void main(String[] args) {
     }
 
     public void run(Scanner scanner) {
-        // napisz swój program tutaj. Do wczytywania danych użyj przekazanego w parametrze scannera
+        List<Integer> list = loadDataFromUser(scanner);
+        printReversedNumberList(list);
+        calculateAndPrintSum(list);
+        printTheBiggestAndTheSmallestNumber(list);
+    }
+
+    private static void printTheBiggestAndTheSmallestNumber(List<Integer> list) {
+        if (!list.isEmpty()) {
```

Comment:
We wszystkich metodach sprawdzasz, czy lista jest pusta. Zamiast tego możesz to zrobić tylko raz w metodzie run